src/PresentationalComponents/Cards/OverviewDashbarCard/OverviewDashbarCard.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import { Card, CardBody, Flex, FlexItem, Text } from '@patternfly/react-core';
+import { DashbarCardTitle } from './components/DashbarCardTitle/DashbarCardTitle';
+import { DashbarCardTagOrIcon } from './components/DashbarCardTagOrIcon/DashbarCardTagOrIcon';
+import { OverviewDashbarCardSupportedTitles } from '../../../AppConstants';
+
+export const OverviewDashbarCard = ({ title, count, onClickFilterByTitle }) => {
+  if (OverviewDashbarCardSupportedTitles[title])
```

Comment:
This feels a bit odd and it is probably not really needed, or what would be the situation when there would be a title passed in that does not exist?

Reply:
In case we render this kind of card with a different title
Because these cards are very specific and handle cards with different titles differently, I have added this condition, like in all of the other components there is a swtich-case to adjust the implementation of the card according to the title given to it

Reply:
Do you think I should remove this condition completely ?

Reply:
After our conversation, I have removed this condition entirely 👍🏼